indexing-service/src/test/java/org/apache/druid/indexing/common/task/IngestionTestBase.java

```diff
@@ -314,7 +314,8 @@ public File getTaskReportsFile()
         );
 
         final TaskToolbox box = new TaskToolbox(
-            new TaskConfig(null, null, null, null, null, false, null, null, null, false, false),
+            new TaskConfig(null, null, null, null, null, false, null, null, null, false, false,
+                           TaskConfig.BatchProcessingMode.CLOSED_SEGMENTS.name()),
```

Comment:
nit: should this use default instead of this explicit value?

Reply:
done in next